drivers/sensors/gps_api.c

```diff
@@ -81,3 +135,26 @@ RV_t gpsInit(void)
 
   return 0;
 }
+
+RV_t gpsRegisterEventCb(gpsEvent_t event, gpsCbFunc_t cb)
+{
+  if (event != GPS_EVENT_LAT_LONG)
```

Comment:
Maybe "if (event >= GPS_EVENT_LAST)"?
Will help to add new events w/o modifying gpsRegisterEventCb().